lost-and-found-kata-solutions/src/test/java/org/eclipse/collections/lostandfoundkata/multireader/MultiReaderCollectionsTest.java

```diff
@@ -31,9 +31,10 @@
 import org.eclipse.collections.impl.factory.Sets;
 import org.eclipse.collections.impl.list.Interval;
 import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.api.Test;
-
+@Disabled
```

Comment:
Hi @jinjin1919 , can you just add annotation Disabled to "multiReaderListFiltering" and not the entire class, so the remaining tests can still run. Thank you.

Reply:
Hi @prathasirisha , sorry about that. I just changed @diabled annotation to the "multiReaderListFiltering" method. instead of the whole class.